lib/country-bounding-boxes.ts

```diff
@@ -0,0 +1,217 @@
+export type CountryBoundingBox = [number, number, number, number];
+
+const countryBoundingBoxData: Record<string, CountryBoundingBox | undefined> = {
```

Comment:
So most of this was pulled from this person who made some creative commons bounding boxes for countries.
https://github.com/sandstrom/country-bounding-boxes

I had to make some adjustments because there was some countries missing in that repo and some of the names in that repo didn't match the names we use in the web app Ex. "United States" vs "USA".

I also had to make some adjustments for countries that cross the 180 longitude line (Russia + Fiji) because the map would just reset to the prime meridian + equator if one of those countries was selected so I just cut off the tiny part of their countries that cross that line, I don't think it's super noticeable. I left Antarctica alone.

Basically this commit is a summary of everywhere I deviated from the publicly available dataset. When data wasn't available I just went to Google Earth and eyeballed it. I tested out all of the countries I eyeballed and I think I did an okay job. https://github.com/PathoTracker/Pathotracker/pull/19/commits/a7c53b7b2191efec70c8894a2a9accedf941cfa1

The map resets it's position if you select two countries that are really far away because it can't fit both of them on the same screen or if you pick two countries really close to longitude 180 and on either side (Ex. Fiji and USA) so there's some stuff we can iterate on here but at least this gives us something to iterate on hahaha

Reply:
This is awesome!